Anagram.java

```diff
@@ -29,22 +28,63 @@ public static void main(String args[]) {
 
 	// Returns true if the two given strings are anagrams, false otherwise.
 	public static boolean isAnagram(String str1, String str2) {
-		// Replace the following statement with your code
-		return false;
+		str1=preProcess(str1);
+		str2=preProcess(str2);
+		boolean check = true;
+		String str2temp = str2;
+		for (int i=0;i<str1.length();i++){
+			check = false;
+			str2= str2temp; 
+			for (int j=0;j<str2.length();j++) {
+				str2temp = "";
+				if (str1.charAt(i) == str2.charAt(j)) {
+					check = true;
```

Comment:
you are using the same variables to override the data over and over again. 
It is bad practice